aten/src/ATen/functorch/BatchRulesDecompositions.cpp

```diff
@@ -160,6 +160,11 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatchedDecomposition, m) {
   OP_DECOMPOSE(linalg_svd);
   OP_DECOMPOSE(linalg_svdvals);
   OP_DECOMPOSE(linalg_tensorinv);
+  OP_DECOMPOSE(linalg_matrix_power);
+  OP_DECOMPOSE(linalg_pinv);
+  OP_DECOMPOSE(linalg_eigh);
+  OP_DECOMPOSE(linalg_vander);
+  OP_DECOMPOSE(linalg_vecdot);
```

Comment:
vecdot, vander, matrix_power seem valid and covered by testing.

For the rest: are they not covered by testing? I don't see the removal of xfails for them

Reply:
I'm not sure, I added these because I was getting some failures I think `aten::linalg_pinv` was needed for one of the other implementations. Trying again, I no longer think that is the case. 

I'm not sure why `TestFunctorchDispatcher.test_unimplemented_batched_registrations_[aten::linalg_pinv]`  is failing but there are no xfails for `py.test test/functorch/test_vmap.py -v  -k 'linalg_pinv'`

```
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_op_has_batch_rule_linalg_pinv_cpu_float32 PASSED                                         [  5%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_op_has_batch_rule_linalg_pinv_hermitian_cpu_float32 PASSED                               [ 11%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_op_has_batch_rule_linalg_pinv_singular_cpu_float32 SKIPPED (test is slow; run with P...) [ 16%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_exhaustive_linalg_pinv_cpu_float32 PASSED                                           [ 22%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_exhaustive_linalg_pinv_hermitian_cpu_float32 PASSED                                 [ 27%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_exhaustive_linalg_pinv_singular_cpu_float32 SKIPPED (test is slow; run with PYT...) [ 33%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_linalg_failure_1D_input_linalg_pinv_cpu_float32 PASSED                              [ 38%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_linalg_failure_1D_input_linalg_pinv_hermitian_cpu_float32 PASSED                    [ 44%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCPU::test_vmap_linalg_failure_1D_input_linalg_pinv_singular_cpu_float32 SKIPPED (test is slow;...) [ 50%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_op_has_batch_rule_linalg_pinv_cuda_float32 PASSED                                       [ 55%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_op_has_batch_rule_linalg_pinv_hermitian_cuda_float32 SKIPPED (no MAGMA library dete...) [ 61%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_op_has_batch_rule_linalg_pinv_singular_cuda_float32 SKIPPED (test is slow; run with...) [ 66%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_exhaustive_linalg_pinv_cuda_float32 PASSED                                         [ 72%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_exhaustive_linalg_pinv_hermitian_cuda_float32 SKIPPED (no MAGMA library detected)  [ 77%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_exhaustive_linalg_pinv_singular_cuda_float32 SKIPPED (test is slow; run with P...) [ 83%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_linalg_failure_1D_input_linalg_pinv_cuda_float32 PASSED                            [ 88%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_linalg_failure_1D_input_linalg_pinv_hermitian_cuda_float32 SKIPPED (no MAGMA l...) [ 94%]
test/functorch/test_vmap.py::TestVmapOperatorsOpInfoCUDA::test_vmap_linalg_failure_1D_input_linalg_pinv_singular_cuda_float32 SKIPPED (test is slo...) [100%]

======================================================= 9 passed, 9 skipped, 3711 deselected in 6.12s ========================================================

```

Reply:
removed the other operators.